radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/transport/udp/NettyUDPTransportImpl.java

```diff
@@ -104,6 +114,7 @@
 		this.controlFactory = controlFactory;
 		this.connectionFactory = connectionFactory;
 		this.priority = config.priority(1000);
+		this.messageBufferSize = config.messageBufferSize(255);
```

Comment:
This is a reasonable default IMO, but it really should be consistent with the TCP transport.